cmd/embedded-cluster/reset.go

```diff
@@ -170,7 +198,7 @@ func newHostInfo(ctx context.Context) (hostInfo, error) {
 		return currentHost, err
 	}
 	// control plane only stff
-	if currentHost.isControlPlane() {
+	if currentHost.Status.Role == "controller" {
```

Comment:
Does this factor in the case where the vendor changed the controller node role name?

Reply:
@ajp-io this specific role is set by k0s, there is only "controller" and "worker"